### PR TITLE
New package: sidplayfp-2.0.1

### PR DESCRIPTION
**Pull request by Zeinok**
Command-line player for libsidplayfp
tested on x86_64

### CODE REVIEW

**Comment by Zeinok** [comment]
eh apparently CI passed but it isn't updating status

https://travis-ci.org/github/void-linux/void-packages/builds/670621820

File: srcpkgs/sidplayfp/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'sidplayfp'
+pkgname=sidplayfp
+version=2.0.1
+revision=1
+build_style=gnu-configure
+hostmakedepends="autoconf pkg-config"
```
Comment (xtraeme) [inline srcpkgs/sidplayfp/template:6]: `autoconf` is not really needed

Comment (Zeinok) [reply srcpkgs/sidplayfp/template]: removed

**Comment by Zeinok** [comment]
CI status is still not updated.
Package successfully built in Travis CI.
https://travis-ci.org/github/void-linux/void-packages/builds/671174946

**Pull request merged**